MyHashSet.java

```diff
@@ -0,0 +1,52 @@
+/*
+	Add Time Complexity : O(1)
+	Remove Time Complexity : O(n)
+	Contains Time Complexity : O(n)
+	
```

Comment:
Please update code to have O(1) for both remove and contains as this is an implementation of Hash Set.